src/lang/abstractSyntaxTree.test.ts

```diff
@@ -1677,17 +1677,12 @@ describe('should recognise callExpresions in binaryExpressions', () => {
 describe('parsing errors', () => {
   it('should return an error when there is a unexpected closed curly brace', async () => {
     const code = `const myVar = startSketchAt([}], %)`
+    const result = parse(code)
 
-    let _theError
-    try {
-      // eslint-disable-next-line
-      let _ = expect(parse(code))
-    } catch (e) {
-      _theError = e
-    }
-    const theError = _theError as any
-    expect(theError).toEqual(
```

Comment:
This `.toEqual` business _was not_ actually checking the inner properties like the Vitest docs said it should 😠 I was able to put any value in and it was passing. So I broke them out to a few explicit value assertions.

Reply:
> WARNING

> A deep equality will not be performed for Error objects. Only the message property of an Error is considered for equality. To customize equality to check properties other than message, use [expect.addEqualityTesters](https://vitest.dev/api/expect.html#expect-addequalitytesters). To test if something was thrown, use [toThrowError](https://vitest.dev/api/expect.html#tothrowerror) assertion.

From vitest docs - but damn that sucks lol